src/python/qeqiskit/backend/backend.py

```diff
@@ -79,6 +82,8 @@ def run_circuit_and_measure(self, circuit, **kwargs):
         ibmq_circuit = circuit.to_qiskit()
         ibmq_circuit.barrier(range(num_qubits))
         ibmq_circuit.measure(range(num_qubits), range(num_qubits))
+        self.number_of_circuits_run += 1
+        self.number_of_jobs_run += 1
```

Comment:
Again, I think it would be preferred to use `super().run_circuit_and_measure(circuit)`

Reply:
Addressed in [d81040](https://github.com/zapatacomputing/qe-qiskit/pull/18/commits/d8104090971fb60f46de371cafe4e6e745c211d3).